src/shared/utils/file-operations.js

```diff
@@ -98,8 +98,18 @@ export function selectAndUploadFile(notebookID, successCallback = () => {}) {
   });
 }
 
-export function makeFormData(notebookID, data, fileName) {
-  const file = valueToFile(data, fileName);
+export function makeFormData(
```

Comment:
this change to `makeFormData` allows you to pass in a file as the data arg. Subject to change.